src/main/java/com/minecolonies/coremod/entity/mobs/aitasks/EntityAIWalkToRandomHuts.java

```diff
@@ -167,6 +174,20 @@ public void tick()
 
         if (this.isEntityAtSiteWithMove(targetBlock, 2))
         {
+            entity.swingArm(Hand.OFF_HAND);
```

Comment:
Why was this change made?

Reply:
ah forgot to add this part, it is for giving raiders a small chance to steal an item when reaching a building to give players some incentive to defend their buildings aswell and not only the citizens (e.g. the extreme of recall all at the townhall and block the exit during raids)

Reply:
This is not going to work
And will cause all kinds of issues.

It is a good idea in concept, but this will cause amazingly bad problems with the RS and citizens.
I for see a bunch of citizens getting stuck because of this, aborting processing or any of a bunch of other types of problems.

So please revert this.

Reply:
I don't mind removing it, ray didnt like it much either.
But it should cause no problems for the RS at all, else any kind of outside intervention like players/hoppers etc will break it

Reply:
yeah, in theory it shouldn't matter. If anything it will expose AI weaknesses more likely.

Reply:
But, I wouldn't add this yet, not without some other balancing (having raiders with items not fight back and try to walk back to camp or similar)